.github/workflows/main.yml

```diff
@@ -193,6 +196,15 @@ jobs:
       - name: forge test all examples
         run: ../.github/scripts/forge-test.sh
         working-directory: examples
+      - name: cargo run -F verify all examples
+        run: ../.github/scripts/run-verify.sh
+        working-directory: examples
+        env:
+          RUST_LOG: "info,risc0_steel=debug,risc0_op_steel=debug"
+          RISC0_DEV_MODE: true
+          L1_RPC_URL: https://eth-mainnet.g.alchemy.com/v2/${{ secrets.ALCHEMY_RISC0_ETH_API_KEY }}
```

Comment:
Here we run the CI against the actual live network, an alternative would be be to spin up a local docker devnet containing at least an L1, L2, and Beacon node